CONTRIBUTING.rst

```diff
@@ -30,6 +36,13 @@ Implement Features
 Look through the GitHub issues for features. Anything tagged with "feature"
 is open to whoever wants to implement it.
 
+
+Improvement requests
+~~~~~~~~~~~~~~~~~~~~
+
+If you see some of the things could be done in a better way, feel free to submit a issue with a improvement suggestion.
```

Comment:
wording suggestion:

If you see room for improvement, please open an issue with a suggestion.